### PR TITLE
Remove Loki

### PR DESCRIPTION
**Pull request by shorim**
**Description**

Changes proposed in this pull request:

- Remove Loki resources.
- Remove logging from components list.
- Adjust docs for Loki removal.
- Add migration script.

**Related issue(s)**

#17637

### CODE REVIEW

File: docs/04-operation-guides/operations/02-install-kyma.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ To deploy Kyma with only specific components, run:
   kyma deploy --components-file {COMPONENTS_FILE_PATH}
   ```
 
-  `{COMPONENTS_FILE_PATH}` is the path to a YAML file containing the desired component list to be installed. In the following example, only eight components are deployed on the cluster:
+  `{COMPONENTS_FILE_PATH}` is the path to a YAML file containing the desired component list to be installed. In the following example, only seven components are deployed on the cluster:
```
Comment (NHingerl) [inline docs/04-operation-guides/operations/02-install-kyma.md:115]: good job for noticing this!!!

**Review by NHingerl — APPROVED** [review]

File: docs/04-operation-guides/operations/02-install-kyma.md (hunk 1 above)
Comment (shorim) [reply docs/04-operation-guides/operations/02-install-kyma.md:115]: Thank you! 😄

File: docs/migration-guide-2.15-2.16.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+---
+Migration Guide 2.15-2.16
+---
+
+Loki has been removed with Kyma 2.16. After upgrading to Kyma 2.16, run the script [2.15-2.16-cleanup-loki.sh](./assets/2.15-2.16-cleanup-loki.sh) to remove Loki.
```
Comment (NHingerl) [inline docs/migration-guide-2.15-2.16.md:6]: or 2.17?

Comment (shorim) [reply docs/migration-guide-2.15-2.16.md:6]: yes, it will be 2.17 now. I will change it

**Review by chrkl — APPROVED** [review]

**Review by ruanxin — APPROVED** [review]

**Pull request merged**